test/connections.js

```diff
@@ -408,6 +408,7 @@ test('relayed connection', async function (t) {
   const socket = b.connect(server.address().publicKey)
 
   server.on('connection', (socket) => {
+    socket.on('error', () => {})
```

Comment:
Sorry, have a question, why would this test give error? Both sides are gracefully doing `socket.end()`

Reply:
The client side doesn't and is now forcefully destroyed on teardown.